src/types/phone.js

```diff
@@ -0,0 +1,22 @@
+const phone = {
+  default: context => {
+    const defaultPhoneRegEx = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/
```

Comment:
I'm so torn on this. I was about to comment that it would be good to add the optional leading `1`, but then I thought about the use case: If you're using obey as a string validator, you're going to want all your phone numbers in the _same_ format. This would allow for 123.123-1234 and 1231231234 and 123-123.1234 and a few other variations. If this were validating content for us to insert into our own DBs, I'd insist on exactly one predictable format. While I love the idea of pre-defining types so others don't have to, I feel like, in this case, we're writing a type that won't be useful to anyone.